scripts/l10n/l10n_helper.py

```
@@ -0,0 +1,241 @@
+                entry = po_file.find(msgid)
+                if not entry:
+                    continue
+                msgstr = msg_mapping[language]
+                if entry.msgstr != msgstr:
+                    entry.msgstr = msgstr
+                    changed = True
+
+            if changed:
+                po_file.save(po_path, newline="\n")
+
+    def fill_missing_translations(self):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)